command/base_helpers.go

```diff
@@ -292,3 +292,16 @@ func parseFlagFile(raw string) (string, error) {
 
 	return raw, nil
 }
+
+func printArgsWarningIfAny(w io.Writer, args []string) {
```

Comment:
This doesn't actually do any printing, how about `generateFlagWarnings`, `generateTrailingFlagWarnings` or similar?